lib/saml20.js

```diff
@@ -97,10 +130,15 @@ exports.create = function(options, callback) {
       // </saml:Attribute>
       var attributeElement = doc.createElementNS(NAMESPACE, 'saml:Attribute');
       attributeElement.setAttribute('Name', prop);
+
+      if (options.includeAttributeNameFormat){
+        attributeElement.setAttribute('NameFormat', getNameFormat(prop));        
```

Comment:
are we handling the case where `prop` is undefined? (I don't know if it's possible)